common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -661,18 +683,47 @@
                 this.showLanguageMenu = true;
 
                 $.each(languages, function(code, label) {
-                    var li = $('<li data-lang-code="' + code + '" />'),
-                        link = $('<button class="control control-lang">' + label + '</button>');
+                    li = HtmlUtils.interpolateHtml(
+                        HtmlUtils.HTML(
+                            '<li data-lang-code="{code}" />'
+                        ),
+                        {
+                            code: code
+                        }
+                    ).toString();
+                    
+                    link = HtmlUtils.interpolateHtml(
+                        HtmlUtils.HTML(
+                            '<button class="control control-lang" aria-pressed="false">{label}</button>'
+                        ),
+                        {
+                            label: label
+                        }
+                    ).toString();
+                    
+                    liObj = $.parseHTML(li);
```

Comment:
The thought behind parsing it back into an HTML object with `$.parseHTML()` was that it's already been made safe, so let's create a safe-ready object for use. FWIW.